toolkit/tools/internal/testutils/testrpms/download-test-rpms.sh

```diff
@@ -6,35 +6,55 @@ CONTAINER_TAG="imagecustomizertestrpms:latest"
 DOCKERFILE_DIR="$SCRIPT_DIR/downloader"
 
 AZURELINUX_2_CONTAINER_IMAGE="mcr.microsoft.com/cbl-mariner/base/core:2.0"
+AZURELINUX_3_CONTAINER_IMAGE="mcr.microsoft.com/azurelinux/base/core:3.0"
 
 IMAGE_VERSION="2.0"
 
-while getopts "t:" flag
+IMAGE_CREATOR="false"
+TOOLS_FILE="$SCRIPT_DIR/tools.tar.gz"
+
+
+while getopts "s:t:" flag
 do
     case "${flag}" in
+        s) IMAGE_CREATOR="$OPTARG";;
         t) IMAGE_VERSION="$OPTARG";;
         h) ;;&
         ?)
-            echo "Usage: download-test-rpms.sh [-t IMAGE_VERSION]"
+            echo "Usage: download-test-rpms.sh [-t IMAGE_VERSION] [-s IMAGE_CREATOR]"
             echo ""
             echo "Args:"
             echo "  -t IMAGE_VERSION   The Azure Image version to download the RPMs for."
+            echo "  -s IMAGE_CREATOR   If set to true, the script will create a tar.gz file with the tools and download the rpms needed to test imagecreator."
             echo "  -h Show help"
             exit 1;;
     esac
 done
 
+
 case "${IMAGE_VERSION}" in
+  3.0)
+    CONTAINER_IMAGE="$AZURELINUX_3_CONTAINER_IMAGE"
+    ;;
   2.0)
     CONTAINER_IMAGE="$AZURELINUX_2_CONTAINER_IMAGE"
-    ;;
+    ;;  
   *)
     echo "error: unsupported Azure Linux version: $IMAGE_VERSION"
     exit 1;;
 esac
 
 set -x
 
+# call the script to create the tools file if requested
+if [ "$IMAGE_CREATOR" = "true" ]; then
+  echo "Creating tools file: $TOOLS_FILE"
+  $SCRIPT_DIR/create-tools-file.sh "$CONTAINER_IMAGE" "$TOOLS_FILE"
```

Comment:
`download-test-rpms.sh` and `create-tools-file.sh` are doing two very different things. So, the scripts should be independent of each other.